db/migrate/20180403161130_devise_create_users.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+      t.string :first_name,         null:false, default: ""
```

Comment:
Don't set a default for first and last name, let the creation fail if they are not specified